services/app-web/src/pages/SubmissionSideNav/SubmissionSideNav.tsx

```diff
@@ -0,0 +1,181 @@
+        if (err instanceof ApolloError) {
+            handleApolloError(err, true)
+        } else {
+            recordJSException(err)
+        }
+        return <GenericErrorPage /> // api failure or protobuf decode failure
+    }
+
+    const { data, formDatas, documentDates } = fetchResult
+    const pkg = data.fetchHealthPlanPackage.pkg
+
+    // fetchHPP returns null if no package is found with the given ID
+    if (!pkg) {
+        return <Error404 />
+    }
+
+    const submissionStatus = pkg.status
+
+    // Check if user is logged in
+    if (!loggedInUser) {
+        return <GenericErrorPage />
+    }
+    const isCMSUser = loggedInUser?.role === 'CMS_USER'
+
+    // CMS Users can't see DRAFT, it's an error
+    if (submissionStatus === 'DRAFT' && isCMSUser) {
+        return <GenericErrorPage />
+    }
+
+    // State users should not see the submission summary page for DRAFT or UNLOCKED, it should redirect them to the edit flow.
+    if (
+        !isCMSUser &&
+        (submissionStatus === 'DRAFT' || submissionStatus === 'UNLOCKED')
+    ) {
+        navigate(`/submissions/${id}/edit/type`)
+    }
+
+    // Current Revision is the last SUBMITTED revision, SubmissionSummary doesn't display data that is currently being edited
+    // Since we've already bounced on DRAFT packages, this _should_ exist.
+    const edge = pkg.revisions.find((rEdge) => rEdge.node.submitInfo)
+    if (!edge) {
+        const errMsg = `No currently submitted revision for this package: ${pkg.id}, programming error. `
+        recordJSException(errMsg)
+        return <GenericErrorPage />
+    }
+    const currentRevision = edge.node
+    const packageData = formDatas[currentRevision.id]
+
+    const outletContext: SideNavOutletContextType = {
+        pkg,
+        currentRevision,
+        packageData,
+        documentDates,
+        user: loggedInUser,
+    }
+
+    return (
+        <div className={styles.background}>
+            <GridContainer className={styles.container}>
+                {showQuestionsAnswers && (
+                    <div className={styles.sideNavContainer}>
+                        <div className={styles.backLink}>
+                            <Link
+                                asCustom={NavLink}
+                                variant="unstyled"
+                                to={{
+                                    pathname: '/dashboard',
+                                }}
+                            >
+                                <svg
+                                    className="usa-icon"
+                                    aria-hidden="true"
+                                    focusable="false"
+                                    role="img"
+                                >
+                                    <use xlinkHref={`${sprite}#arrow_back`} />
+                                </svg>
+                                {loggedInUser?.__typename === 'StateUser' ? (
```

Comment:
🎨 style nit: The blue on these links look wrong now. Maybe be nested scss issue. Let me know if you want to look together 
![Screenshot 2023-02-03 at 4 28 53 PM](https://user-images.githubusercontent.com/10750442/216723146-7f59b785-b4b6-414e-81da-333690f7c242.png)

Reply:
Nice catch. I check val to make sure it was just my branch, but looks like the link color is also incorrect there too.

![Screenshot 2023-02-06 at 11 24 07 AM](https://user-images.githubusercontent.com/98117700/217027430-7b272826-1465-4f0f-a324-a16fe57d5e07.png)

Reply:
Looks like the fix was to remove `variant="unstyled"` attribute from the `Link` component.

Reply:
Yeah these discrepancies with styling is something I was noticing too with the USWDS work. Plus also feels like the unstyled variant shouldnt override custom css on a component passed in by user 🤔 that might be something worth changing in react-uswds.

Thanks for making the change!